bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/ScriptEngineManager.java

```diff
@@ -16,47 +16,48 @@
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.automation.module.script.internal.provider.ScriptModuleTypeProvider;
 
 /**
+ * The ScriptEngineManager provides the ability to load and unload scripts.
  *
  * @author Simon Merschjohann - Initial contribution
+ * @author Scott Rushworth - changed parameter names when implementing {@link ScriptModuleTypeProvider}
  */
 @NonNullByDefault
 public interface ScriptEngineManager {
 
     /**
-     * Checks if a given fileExtension is supported
+     * Creates a new ScriptEngine used to execute scripts, ScriptActions or ScriptConditions
      *
-     * @param fileExtension
-     * @return true if supported
+     * @param scriptType       a file extension (script) or MimeType (ScriptAction or ScriptCondition)
```

Comment:
Please excuse me, it is really not your fault, but perhaps you can correct it...
There is something wrong with the IDE setup, the formatter should be selected but it seems there is a bug.

Go to your IDE settings, search for formatter, select the Java formatter, choose another formatter then "ESH", and then switch back to "ESH". Apply & close, reformat this file...

There should be no spaces after e.g. `scriptType` and its description.

Reply:
Done